### PR TITLE
Promote OME2021 meeting event

### PR DESCRIPTION
**Pull request by sbesson**
As discussed this morning this makes the community event page more accessible via a new entry in the events page as well as a top-level banner

### CODE REVIEW

**Comment by pwalczysko** [comment]
I kind of like the banner which was put up for the OME 2020 https://user-images.githubusercontent.com/1644105/78772246-6201ff80-7989-11ea-841b-5c9fc9eb3761.png - it attracts much better the attention. It was established in the PR https://github.com/ome/www.openmicroscopy.org/pull/352
At first look, when I built the present PR, https://snoopycrimecop.github.io/www.openmicroscopy.org/ I thought the red line on the top is the old SV announcement (it always used to be SV).
Maybe we could consider to rewrite the https://github.com/ome/www.openmicroscopy.org/pull/352 commits so that we have a better banner such as last year ?

Present state

Screenshot 2021-04-20 at 17 15 12

File: events/index.html [hunk 1]
```diff
@@ -9,6 +9,12 @@
         <!-- begin Events -->
         <hr class="whitespace">
         <div id="community-posts" class="row">
+            <div class="small-12 medium-8 medium-offset-2 columns">
+                <h6><i class="fa fa-calendar"></i> June 08-11, 2021</h6>
+                <h2><a href="ome-community-meeting-2021">2021 OME Community Meeting</a></h2>
+                <p class="card-caption">The 2021 OME Community Meeting will be held online from June 08th to June 11th 2021.</p>
```
Comment (pwalczysko) [inline events/index.html:15]:
```suggestion
                <p class="card-caption">The 2021 OME Community Meeting will be held online from the 8th to the 11th of June 2021.</p>
```

File: index.html [hunk 2]
```diff
@@ -51,8 +51,9 @@
         
         <section class="announcement bg-ome-red">
             <span>
-            Security Update: On 18/03/21 we released version 5.9.1 of OMERO.web
-            containing a few security fixes. Upgrading is highly recommended.
+            Save The Date: The 2021 OME Community Meeting will be held
+            online between June 08-11 2021. See <a class="styled-link" href="{{ site.baseurl }}/events/ome-community-meeting-2021/">here</a> for
```
Comment (pwalczysko) [inline index.html:55]:
```suggestion
            online between June 08-11 2021. See <a class="styled-link" href="{{ site.baseurl }}/events/ome-community-meeting-2021/">preliminary programme</a> for
```

**Comment by sbesson** [comment]
Last commits deployed at https://snoopycrimecop.github.io/www.openmicroscopy.org/

**Comment by pwalczysko** [comment]
@sbesson thank you. LGTM.

**Review by pwalczysko — APPROVED** [review]

**Comment by sbesson** [comment]
Thanks @pwalczysko. Tagging and deploying to start linking to the programme and we will start filling speakers in the upcoming days.

**Pull request merged**